neo/VM/ExecutionEngine.py

```diff
@@ -747,6 +747,30 @@ def ExecuteOp(self, opcode, context):
 
                 estack.PushT(Struct(items))
 
+            elif opcode == APPEND:
+                newItem = estack.Pop()
+
+                if type(newItem) is Struct:
+                    newItem = newItem.Clone()
+
+                arrItem = estack.Pop()
+
+                if not arrItem.IsArray:
+                    self._VMState |= VMState.FAULT
+                    return
+
+                arr = arrItem.GetArray()
+
+                arr.append(newItem)
+
+            elif opcode == REVERSE:
+
```

Comment:
no new line after if

Reply:
I'd say this does not belong to this issue because this commit is from merging the development branch into this branch..